CHANGELOG.md

```diff
@@ -4,6 +4,34 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.17.0
+
+This release standardizes our SDKs to use the MIT License, which is our [standard license for Sentry SDKs](https://open.sentry.io/licensing/). We were previously using the BSD 3-Clause License in `@sentry/browser`,`@sentry/core`, `@sentry/gatsby`, `@sentry/hub`, `@sentry/integrations`, `@sentry/node`, `@sentry/react`, `@sentry/types`, `@sentry/typescript`, and `@sentry/utils`.
+
+This release also updates the behaviour of [`tracingOrigins`](https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/#tracingorigins) to no longer affect span creation. `tracingOrigins` will only affect if `sentry-trace` and `baggage` headers are attached to outgoing requests. To control span creation, use the [`shouldCreateSpanForRequest`](https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/#shouldcreatespanforrequest) option.
+
+- chore: Standardize SDKs on MIT License (#5993)
+- feat(nextjs): Add Next 13 to peer dependencies and integration tests (#6042)
```

Comment:
Should we add a sentence about Next 13 support? (i.e. that it is currently still limited IIUC)

Reply:
I thought about it - but figured that the changelog item should be enough to show that we unblock users.